pallets/omnipool/README.md

```diff
@@ -0,0 +1,51 @@
+# Omnipool pallet
+
+Omnipool implementation
+
+## Overview
+
+Omnipool is type of AMM where all assets are pooled together into one single pool.
+
+Liquidity provider can provide any aset of their choice to the Omnipool and in return 

```

Comment:
asset*

(let me know if we do not nitpick typos, I do not want to do it unnecessarily :))

Reply:
btw you can add a suggestion in the comment which can be directly committed. 

i would suggest to do that, so i can just click "commit change".